libfabric.spec

```diff
@@ -15,6 +15,7 @@ Url: https://www.github.com/ofiwg/libfabric
 Source: https://github.com/ofiwg/%{name}/archive/v%{version}.tar.gz
 Patch0: https://github.com/ofiwg/libfabric/compare/v1.9.0...1b8ed7876204692fd95b07df8cba21683707e5dc.patch
 Patch1: https://github.com/ofiwg/libfabric/compare/1b8ed7876204692fd95b07df8cba21683707e5dc...955f3a07dd011fb1dbfa6b6c772ada03d5af320e.patch
+Patch2: https://github.com/ofiwg/libfabric/compare/955f3a07dd011fb1dbfa6b6c772ada03d5af320e...8af3c112bfce155eb04218bef656f58f3609ce19.patch
```

Comment:
FWIW, we probably want to refrain from just piling on the version bumps as new patches and instead just regenerate `Patch0` to span the whole jump.

This is because I am starting to point external folks (OPA and A21 teams) at this package so keeping the file as clean and straightforward as possible would be good.  Will hopefully keep external questions and/or scepticism down.